packages/react/src/components/Button/Button.stories.tsx

```diff
@@ -70,88 +49,194 @@ export const Preview: Story = {
   argTypes: {
     icon: {
       control: 'select',
-      options: Object.keys(icons),
+      options: Object.keys(akselIcons),
     },
   },
 };
 
-// This is an example of composed story
-export const Variants: Story = {
-  render: VariantsTemplate,
-  decorators: [
-    (Story) => (
-      <Stack>
-        <Story />
-      </Stack>
-    ),
-  ],
-};
+export const UsedAsLink: StoryFn<typeof Button> = () => (
+  <Button
+    as='a'
+    icon={<akselIcons.PersonChatIcon />}
+    iconPlacement='right'
+    target='_blank'
+    href='https://www.designsystemet.no'
+  >
+    Bli med på Slack
```

Comment:
Maybe change the text here to what the link actually is?
Not a big issue, but would be nice.

Reply:
Change the text to `Gå til Designsystemet.no`?